### DISCUSSION TITLE
collection.not used against normal URL list pulls?

### DISCUSSION THREAD

**Discussion by DreamStatic** [meisnate12/Plex-Meta-Manager#270] [Q&A] [answered]
I got to thinking today about this "collection.not" filter option for some queries. Would it be possible to utilize something along those lines to say remove movies from a URL list pull? 

Here is an example as I doubt I am making any sense. lol. 

I have a list on Trakt of movies I am interested in for a movie night with the better half. Some of these movies are older and some are upcoming in the list. As I add movies to my library, they automatically show up in this collection. I would like to not show any that we have already watched. I have a smart collection named "Watched" that includes any movie that has >0 plays. Be cool if when it pulls the Trakt list, it also pulls the Watched collection as a way to not load those movies into the Trakt Interested collection.

Is there anything already in place for something like this? Today is my first day with Plex Meta Manager so still have a lot to learn.

**Comment by meisnate12** [comment, marked as answer]
lol i so didn't get it at first so I appreciate the example I guess what you're really looking for is to have `plays` as a collection filter I could add that in

**Reply by DreamStatic** [reply]
That would be awesome. Thank you so much.

**Reply by meisnate12** [reply]
plays has been added as a collection filter

**Reply by DreamStatic** [reply]
Most awesome! Thank you so much!!
